pinecone/core/utils/__init__.py

```diff
@@ -70,6 +72,37 @@ def get_user_agent():
     return user_agent
 
 
+def dict_to_proto_struct(d: dict) -> Struct:
```

Comment:
@rajat08 This definitions breaks the import since Struct is not defined for non-grpc client